cronjob_update_build_run.sh

```diff
@@ -0,0 +1,37 @@
+repo="https://github.com/FelixStegmeier/wicked_migration_server.git"
+server_dir="./wicked_migration_server"
+migration_dir="$server_dir/wicked_migration_server"
+
+git_has_changes() {
+	main=$(git rev-parse main)
+	origin_main=$(git rev-parse origin/main)
+	[ $main != $origin_main ]
+}
+
+if [ ! -d $server_dir ]; then
+	echo "creating $server_dir"
+	mkdir $server_dir
+fi
+
+if [ ! -d $migration_dir ]; then
+	echo "cloning git repo"
+	git clone $repo $migration_dir
+	cargo build --release --manifest-path $migration_dir/Cargo.toml --target-dir $server_dir
+	cp -r $migration_dir/static $server_dir/static
+fi
+
+if [ "$git_has_changes" ]; then
```

Comment:
This doesn't actually call the function, it just looks for a variable that is never defined, so it will never be true.